Add the Feast of the Exaltation of the Holy Cross to general.js

tukusejssirs opened the pull request:
This issue fixes #11.

jarosz commented:
I'd rather merge romcal-es6 to master first, and ask you to create this request again against new master branch.

tukusejssirs commented:
I’d rather do it in reverse order. This won’t break the API, therefore it should be merged before `v1.3.0` release. Also I think that any and all merges to `master` should be a new release (at least a minor).

cc: @emagnier

emagnier commented on src/locales/fr.js:
```diff
@@ -63,7 +63,7 @@ export default {
     "presentationOfTheLord": "Présentation de Jésus au Temple",
     "sacredHeartOfJesus": "Sacré-Cœur de Jésus",
-    "theExaltationOfTheHolyCross": "The Exaltation of the Holy Cross",
+    "theExaltationOfTheHolyCross": "Exaltation de la Sainte-Croix",
```
After verification in a missal and the [francophone bishop association](https://www.aelf.org/calendrier/romain/2019/09) website, I found that in French we don't literally translate it, but uses instead `La Croix Glorieuse`.

tukusejssirs commented:
@emagnier, I have fixed the localisation of `theExaltationOfTheHolyCross` in `fr.js`.

I have also added another commit (in case you’d like to omit it) with replacing all apostrophes with the typographic form of it (i.e. ’).
